nasa-gibs/worldview
# WV-2494: Add 4 GOES Fire Temp and Dust layers

## Description

Fixes WV-2494 & WV-3156.

Added four GOES-West and GOES-East Dust and Fire Temperature layers.

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-161.70117033290808,8.579848214391191,-58.047064826763396,55.749506467187445&z=4&ics=true&ici=5&icd=10&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,GOES-West_ABI_GeoColor(hidden),GOES-East_ABI_GeoColor(hidden),GOES-West_ABI_FireTemp(hidden),GOES-East_ABI_FireTemp(hidden),GOES-West_ABI_Dust(hidden),GOES-East_ABI_Dust,VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-08-13-T14%3A39%3A41Z)
2. Check to make sure imagery displays, layer descriptions read well, preview images show, and that you can take image snapshots.
@nasa-gibs/worldview
